js/templates/buyWizard.html

```diff
@@ -28,10 +28,12 @@ <h3 class="custCol-text padding30"><%= polyglot.t('CheckExchanges') %> </h3>
         </div>
         <div class="txt-bright pad20">
           <div class="txt-center row10 textSize20px custCol-text">
-            <%= polyglot.t('buyFlow.SendBTCtoAddress', {amount: '<span class="js-buyWizardDetailsTotalBTC"></span>'}) %>
-          </div>
-          <div class="txt-center row10">
-            <a class="textSize16px js-buyWizardPayLink custCol-text"><span class="js-buyWizardPayURL"></span></a>
+            <%=
+              polyglot.t('buyFlow.SendBTCtoAddress', {
+                amount: '<span class="js-buyWizardDetailsTotalBTC"></span>',
+                recipient: '<a class="textSize16px js-buyWizardPayLink custCol-text"><span class="js-buyWizardPayURL"></span></a>'
```

Comment:
Hmm... They can't control any of the markup, they only control the position. Since we're talking about an inline element here, I'm not really sure what damage they could do by altering it's position within the parent div.

It's just a matter of us devs being mindful that the link can end up anywhere in the sentence, so we shouldn't be doing stuff like assuming it will fall into a certain place and putting margin specifically on one side.

Besides, how is making the `recipient` positionable any different than what was already in place with the `amount`?